setup.py

```diff
@@ -2667,7 +2673,7 @@ def copy_scripts(self):
                 newfilename = filename + fullversion
             else:
                 newfilename = filename + minoronly
-            log.info('renaming %s to %s', filename, newfilename)
+            log.info(f'renaming {filename} to {newfilename}')
```

Comment:
With your change, I no longer see this message. IMO you should mimick distutils setup() function which sets the logging thredshold to INFO: 

```
  File "/home/vstinner/python/main/./setup.py", line 2698, in main
    setup(# PyPI Metadata (PEP 301)
  File "/home/vstinner/python/main/Lib/distutils/core.py", line 134, in setup
    ok = dist.parse_command_line()
  File "/home/vstinner/python/main/Lib/distutils/dist.py", line 477, in parse_command_line
    log.set_verbosity(self.verbose)
  File "/home/vstinner/python/main/Lib/distutils/log.py", line 78, in set_verbosity
    set_threshold(INFO)
```

To see the message on the main branch:

```
$ rm -rf build/scripts-3.11/
$ make
(...)
renaming build/scripts-3.11/pydoc3 to build/scripts-3.11/pydoc3.11
renaming build/scripts-3.11/idle3 to build/scripts-3.11/idle3.11
renaming build/scripts-3.11/2to3 to build/scripts-3.11/2to3-3.11
```

Reply:
Thank's nice catch :)